### PR TITLE
Blockchain API transaction lookup

### PR DESCRIPTION
**Pull request by Crypt-iQ**

### CODE REVIEW

**Comment by Giszmo** [comment]
"nsa committed 3 hours ago" ... files changed: binary ... is this a joke?

**Comment by kristovatlas** [comment]
lol

**Comment by kristovatlas** [comment]
If there's a real need to lookup the result of transactions or address balances, I would start with using bitcoind's RPC interface, which is much more private and faster. Barring that, have a list of remote APIs and pick a random one for each lookup.

**Comment by kristovatlas** [comment]
https://en.bitcoin.it/wiki/Bitcoin-JSON-RPC-Client

**Comment by kristovatlas** [comment]
If you don't want to make bitcoind a dependency, you could also use BitcoinJ. https://en.bitcoin.it/wiki/BitCoinJ

**Comment by DanielKrawisz** [comment]
Hahaha, we definitely cannot have binary files in here! Please make a new commit for this pull request without the binary files included.

File: shuffler/src/main/java/com/shuffle/bitcoin/Coin.java [hunk 1]
```diff
@@ -3,6 +3,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Queue;
+import org.bitcoinj.core.Transaction;
```
Comment (DanielKrawisz) [inline shuffler/src/main/java/com/shuffle/bitcoin/Coin.java]: The interfaces shouldn't know anything about the underlying implementation, so there shouldn't ever be any imports like this. And it's not actually being used either so it should be removed.

File: shuffler/src/main/java/com/shuffle/bitcoin/Bitcoin.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.shuffle.bitcoin;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.Queue;
+
+abstract public class Bitcoin implements Coin {
+	
+	String USER_AGENT = "Chrome/5.0";
```
Comment (DanielKrawisz) [inline shuffler/src/main/java/com/shuffle/bitcoin/Bitcoin.java]: I don't think this needs to be there now, does it?

**Pull request merged**